### PR TITLE
starter work on incorporating the deno kernel

### PR DESCRIPTION
**Pull request by rgbkrk**
## Describe your changes

This is the _start_ of including the deno kernel. I'd love feedback on how we carry this out as I imagine we're going to want to migrate from this jupyter branch of deno, followed by `main`, then onto targeting the stable 1.37 release (coming out on 9/18). I'd like to have this dev branch available for us to evaluate in advance of the launch.

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I am able to build images locally
- [x] Has the issue it resolves been discussed with maintainers?

### CODE REVIEW

File: docker-bake.hcl [hunk 1]
```diff
@@ -27,6 +27,21 @@ variable "TIMESTAMP" {
...
+group "deno" {
+  targets = [
+    "deno_1_37-dev",
+  ]
+}
+
+# Group of Deno 1.37 targets
+group "deno_1_37-dev" {
+  targets = [
+    "deno_base_1_37-dev",
```
Comment (robwittman) [inline docker-bake.hcl:40]: This target would need to also be added to this file. something like 
```hcl 
target "deno_base_1_37-dev" {
  inherits = ["base"]
  context = "deno/base/1.37-dev"
  tags = [
    "ghcr.io/noteable-io/kernel-deno-1.37-base:${TAG}",
    notequal("", HANDLE) ? "ghcr.io/noteable-io/kernel-deno-1.37-base:${HANDLE}" : ""
  ]
}
```
Just for edification:
- `target` references an object that builds an actual artifact
- `group` is simply a helper to execute multiple targets at once

Comment (rgbkrk) [reply docker-bake.hcl:40]: Thanks. I clearly didn't copy the rust kernel images enough. 😆

File: Taskfile.deno.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+version: 3
+
+vars:
+  NBL_DENO_VERSION: 1.37-dev
```
Comment (shouples) [inline Taskfile.deno.yaml:4]: Should this just be `1.37` since the tag is set to `dev` by default for these Taskfile builds?

**Review by shouples — APPROVED** [review]
Tested locally with our stack and it's looking good! 🚀

**Review by robwittman — APPROVED** [review]

**Pull request merged**